core/src/main/java/org/apache/brooklyn/util/core/config/ConfigBag.java

```diff
@@ -367,6 +367,22 @@ public Object getStringKey(String key) {
         }
     }
 
+    /** Puts into this bag the raw value for the given key in the given bag, if it was present. */
+    @Beta
+    public <T> ConfigBag copyRaw(ConfigBag source, ConfigKey<T> key) {
```

Comment:
would `copy(...)` be a better name?  `Raw` makes it feel ugly when actually i think this is quite a nice pattern for dealing with transferring values without resolving them